### ISSUE TITLE
Rocket Motors

### ISSUE THREAD

**Issue by MrkWKD** [studio-minus/ppg-bugs-and-requests#2117]
**Feature description**
A small single use solid fuel rocket motor with less thrust than a small thruster but more than a firework with adjustable fuel load (timed delay) and *almost* harmless exhaust.

**Feature purpose**
Useful for making smaller disposable rocket propelled stuff like model rockets, small scape propelled explosives (rpgs, javelins ect), just anything small that needs disposable thrust with decent power and little weight.

**Comment by TeslaCoil9001** [comment]
It would also be good because the Ion thruster is kind of crappy and we need something less deadly but still powerful. I really like this suggestion and it could add a lot!

**Comment by DindinYT37** [comment]
> It would also be good because the Ion thruster is kind of crappy and we need something less deadly but still powerful. I really like this suggestion and it could add a lot!

How are Ion Thrusters "crappy"? They're literally just weaker thrusters that cause no damage to other machinery...

**Comment by YourLocalPlonker** [comment]
Yeah a solid rocket engine would be cool. It would make my air to air missiles more realistic. (currently I am using the normal thrusters which are actually liquid rocket engines)

**Comment by mestiez** [comment]
Idk about single use

**Comment by MrkWKD** [comment]
well its your game its up to you lol

**Comment by Drogo-man** [comment]
You solid rocket fuel boosters?
